jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/service/SummaryService.java

```diff
@@ -0,0 +1,474 @@
+    }
+    if (imageSummary.getImageSha() != null) {
+      logger.info("    * SHA: %s", imageSummary.getImageSha());
+    }
+  }
+
+  private void printBanner() {
+    logger.info(" __ / / //_/ / / / _ )/ __/");
+    logger.info("/ // / ,< / /_/ / _  / _/  ");
+    logger.info("\\___/_/|_|\\____/____/___/  \n");
+    logger.info(DASHED_LINE);
+    logger.info("      SUMMARY");
+    logger.info(DASHED_LINE);
+  }
+
+  private void printBuildSummary(Summary summary) {
+    if (summary.getBuildStrategy() != null) {
+      logger.info("Build Strategy : %s", summary.getBuildStrategy());
+    }
+    if (summary.getGeneratorsApplied() != null && !summary.getGeneratorsApplied().isEmpty()) {
+      logger.info("Generators applied: [%s]",  String.join(",", summary.getGeneratorsApplied()));
+    }
+    if (summary.getOpenShiftBuildConfigName() != null) {
+      logger.info("Build config: %s", summary.getOpenShiftBuildConfigName());
+    }
+    logger.info("");
+  }
+
+  private void printResourceSummary(Summary summary, File baseDir) {
+    if (summary.getGeneratedResourceFiles() != null && !summary.getGeneratedResourceFiles().isEmpty()) {
+      if (summary.getEnrichersApplied() != null && summary.getEnrichersApplied().size() < 20) {
+        logger.info("Enrichers applied: [%s]", String.join(",", summary.getEnrichersApplied()));
+      }
+      logger.info("Generated resources:");
+      List<String> generatedFilesResourcePaths = summary.getGeneratedResourceFiles().stream()
+          .map(File::getAbsolutePath)
+          .map(p -> FileUtil.getRelativeFilePath(baseDir.getAbsolutePath(), p))
+          .collect(Collectors.toList());
+      logList(generatedFilesResourcePaths);
+    }
+    if (summary.getAggregateResourceFile() != null) {
+      logger.info(LIST_ELEMENT, FileUtil.getRelativeFilePath(baseDir.getAbsolutePath(), summary.getAggregateResourceFile().getAbsolutePath()));
+    }
+    logger.info("");
+  }
+
+  private void printApplySummary(Summary summary) {
+    if (StringUtils.isNotBlank(summary.getAppliedClusterUrl())) {
+      logger.info("Applied resources from %s", summary.getAppliedClusterUrl());
+      printKubernetesResourceSummary(summary.getAppliedKubernetesResources());
+      logger.info("");
+    }
+  }
+
+  private void printUndeploySummary(Summary summary) {
+    if (StringUtils.isNotBlank(summary.getUndeployedClusterUrl())) {
+      logger.info("Undeployed resources from %s", summary.getUndeployedClusterUrl());
+      printKubernetesResourceSummary(summary.getDeletedKubernetesResources());
+      logger.info("");
+    }
+  }
+
+  private void printKubernetesResourceSummary(List<KubernetesResourceSummary> kubernetesResourceSummaries) {
+    if (kubernetesResourceSummaries != null && !kubernetesResourceSummaries.isEmpty()) {
+      for (KubernetesResourceSummary kubernetesResourceSummary : kubernetesResourceSummaries) {
+        logger.info(LIST_ELEMENT, kubernetesResourceSummary.getResourceName());
```

Comment:
This is what I have
```
[INFO] k8s: Applied resources from https://192.168.49.2:8443/
[INFO] k8s:  - quarkus
[INFO] k8s:    * v1 Service
[INFO] k8s:    * Namespace: default
[INFO] k8s:  - quarkus
[INFO] k8s:    * apps/v1 Deployment
[INFO] k8s:    * Namespace: default
```
I would try to be minimalist here and only display kind/name:
```
[INFO] k8s: Applied resources from https://192.168.49.2:8443/
[INFO] k8s:  [ Service/quarkus, Deployment/quarkus]
```
What do you think ?

Reply:
okay, I've modified it as per your suggestion. Where should be specify deployed namespace?

Reply:
I am unsure: are we always applying resources in the same namespace?